### ISSUE TITLE
Add functionality to wait for document.ready?

### ISSUE THREAD

**Issue by W1zzardTPU** [muicss/loadjs#55]
What do you think about adding an internal dependency called "document", that's triggered by document.ready?

I faced random issues with jquery dependencies until I realized my jQuery code now got executed before the DOM was ready. Clearly a coding bug on my side, but it got me thinking.

**Comment by amorey** [comment]
DOM timing with jQuery can definitely get tricky. I'm hesitant to add a jQuery-specific feature but I'll think about it some more. In case it helps you can combine your loadjs calls into one:
```javascript
loadjs('/path/to/jquery.js', function() {
  $(function() {loadjs.done('jquery');});
});

// execute code after jQuery document.ready
loadjs.ready('jquery', function() {
});
```

**Comment by W1zzardTPU** [comment]
Yeah, that's what I'll use, was just waiting for the new release :)
